config/initializers/about_page.rb

```diff
@@ -9,7 +9,7 @@
   config.fedora           = AboutPage::Fedora.new(ActiveFedora.fedora.connection)
   config.solr             = AboutPage::Solr.new(ActiveFedora.solr.conn, :numDocs => 1)
   config.database         = Avalon::About::Database.new(User)
-  config.mediainfo        = Avalon::About::MediaInfo.new(:version => '>=0.7.59')
+  config.mediainfo        = Avalon::About::MediaInfo.new(path: Settings.mediainfo.path)
```

Comment:
Does it make sense to keep this or can it be removed?

Reply:
Maybe it isn't needed? My thought was that since we still have to require installation of MediaInfo for ActiveEncode (and possibly other gems?), it makes sense to keep it around on the About page to show where and what version is installed.